### DISCUSSION TITLE
Add decdec Twig filter

### DISCUSSION THREAD

**Discussion by alexmglover** [craftcms/cms#8730] [Ideas]
There is currently an `encenc` Twig filter, so it'd be great to be able to use `decdec` as well.

**Comment by brandonkelly** [comment]
What’s the use case? Generally you’d only need to decode a string from the back-end.

**Comment by alexmglover** [comment]
In my case, I was using encenc to build a url to an asset then was wanting to do `{{ assetId | decdec | download }}` but I ended up writing it into my download function.
